server.js

```diff
@@ -105,6 +105,18 @@ app.get('/robots.txt', (req, res)=>{
 	return res.sendFile(`${__dirname}/robots.txt`);
 });
 
+//Home page
+app.get('/', async (req, res, next)=>{
+	req.welcomeText = welcomeText;
+	return next();
+});
+
+//Changelog page
+app.get('/changelog', async (req, res, next)=>{
+	req.changelogText = changelogText;
+	return next();
+});
+
```

Comment:
Let's see... In `Homebrew.jsx` we just use changelogText to create a barebones `brew` object that gets passed into a share page. How about we just create that brew object here instead, and clean up the number of "special-case" props below?

Reply:
> Sounds remarkably sensible, something like:
>
> ```
> app.get('/changelog', async (req, res, next)=>{
>	req.text = changelogText;
>	return next();
> });
> ```

I would just use the minimal brew you already made. No need for a separate `req.text`.  By making a `brew` we no longer use the `changeLogText` prop at all and we can just remove it.

Reply:
And yes, we can just pass in `this.props.brew.text` to the Homepage route in Homebrew.jsx since Homepage is weird and doesn't accept a full brew (to be fixed later as you say...).